_posts/2019-06-09-deploy-kub-to-gke.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Deploying a Kub App on GKE
+# permalink: /guides/deploy-kubernetics-to-gke
+logoUrl: /assets/img/guides/kubernetics_image.png
+excerpt: Before we can deploy a dockerized app, we need to first create one. For the purposes of this guide we will create a basic Node.js app that responds to requests on port 8080.
+categories: [guides]
+tags: [Kubernetics, GKE]
+toc: true
+---
+
+* TOC 
+{:toc}
```

Comment:
Might have to use a plugin to support TOC with Markdown: https://github.com/toshimaru/jekyll-toc. AsciiDoc has native TOC support of some sort...